src/Avalonia.FuncUI.DSL/ScrollViewer.fs

```diff
@@ -17,15 +17,15 @@ module ScrollViewer  =
     type ScrollViewer  with            
 
         /// <summary>
-        /// Gets or sets the vertical scrollbar visibility.
+        /// Sets the vertical scrollbar visibility.
         /// </summary>
-        static member verticalScrollBarVisibility<'t when 't :> ScrollViewer>(value: ScrollBarVisibility) : IAttr<'t> =
+        static member verticalScrollBarVisibility<'t when 't :> Control>(value: ScrollBarVisibility) : IAttr<'t> =
```

Comment:
why?

Reply:
Ohh, missed that. I guess it's a attached property and should be an extension ob control

Reply:
Yep, it's attached. Sometimes you don't have the scrollviewer in your code but you still want to hide it.

Reply:
I'll change that on master with my next PR.